src/isomorphic/hooks/ReactComponentTreeDevtool.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactComponentTreeDevtool
+ */
+'use strict';
+
+// TODO remove this proxy when RN/www gets updated
```

Comment:
If we are trying to keep it backward compatible, we also have to allow old method names (addDevtool, removeDevtool).